etc/conf.templates/distributed/10-ingress.conf.template

```diff
@@ -293,7 +293,12 @@ ingress.kafka.metadata.maxsize = 900000
 //
 // Identification of Ingress Metadata Update endpoint source
 //
-#ingress.metadata.update = 
+#ingress.metadata.update =
+
+//
+// Identification of Ingress Metadata Update endpoint source when doing a delta update of attributes
+//
+#ingress.metadata.update.delta =
```

Comment:
This is not a configuration key

Reply:
Neither `ingress.metadata.update`, removing both.